.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -24,6 +24,8 @@ Please delete these comment blocks before submitting the issue.
 
 IMPORTANT!!!
 
+PLEASE CHECK IF THIS ISSUE HAS BEEN REPORTED PREVIOUSLY USING SEARCH!!!
```

Comment:
I think there is a better idea. We should rather point people to the existing but not wildly know  "similar feature" functionality. When you start creating the issue and start typing description of the issue - you get "x similar issues" appear next to the "create issue" button. When you click it you will get ..... a list of similar issues... Maybe we should drag people's attention to this - it's rather natural (but I think Github Failed in making it visible enough). 


![Screenshot from 2020-04-21 16-56-43](https://user-images.githubusercontent.com/595491/79881321-5387ff00-83f1-11ea-82ed-6a7d2d93811e.png)


![Screenshot from 2020-04-21 17-00-13](https://user-images.githubusercontent.com/595491/79881523-9944c780-83f1-11ea-8da2-778c8509f611.png)

Reply:
From my experience this suggestion are not state of work in terms of recommendation engine :<

Reply:
From my experience this suggestion are not state of work in terms of recommendation engine :<

Reply:
:D. Maybe. I have rather good experiences from other projects where I tried to open issue.  I think the workflow is much more probable to happen for people. Searching moves you out from what you wanted to do (i.e. create an issue). I think from the user experience the "similar issues" are much more natural than "search first" and "create issue later". Maybe we can suggest both ?

Reply:
> Maybe we can suggest both ?
 I think so